kitchen.vagrant.yml

```diff
@@ -1,71 +1,24 @@
-<%
-require 'time';
-%>
 ---
 driver:
   name: vagrant
   customize:
     natdnsproxy1: "on"
     natdnshostresolver1: "on"
 
-provisioner:
-  name: puppet_apply
-  sudo: true
-  require_chef_for_busser: false
-  manifests_path: ./spec/puppet/manifests
-  modules_path: ./spec/puppet/modules
-  hiera_config_path: ./spec/puppet/hiera.yaml
-  hiera_data_path: ./spec/puppet/hiera
-  resolve_with_librarian_puppet: true
-  ignore_spec_fixtures: true
-
-verifier:
-  name: inspec
-  sudo: true
-  reporter:
-    - cli
-    - json:spec/results/%{platform}_%{suite}-<%= Time.now.iso8601 %>.json
-  inspec_tests:
-    - name: RedHat Enterprise Linux 7 STIG
-      path: .
-      #branch: "v2.6"
-  #controls:
-  #  - V-72309
-  input_files:
-    - inputs.example.yml
-  inputs:
-    disable_slow_controls: true
```

Comment:
Again, we may want to review these defaults to ensure that:
a. we set sensible default with respect to the profile and STIG
b. that we create defaults that are sensible for our CICD

Reply:
The `inputs.example.yml` sets `disable_slow_controls: true` as well, and the `inputs.example.yml` is still set in the default `kitchen.yml`. Will you want to change the value in `inputs.example.yml` to false as well or will that stay as is?